generator/src/main/java/com/scottlogic/deg/generator/GenerateTestCases.java

```diff
@@ -116,6 +124,15 @@ public void run() {
         }
     }
 
+    private DataSetWriter getWriter(){
```

Comment:
Hi, is there a reason for initialising outputWriter before checking if tracing is enabled? Just thinking that we might save ourselves a little time not having to garbage collect it later.

Reply:
The CSV writer is initialised before the check as its used for both tracing enabled AND disabled outputs.